### ISSUE TITLE
How to remove event listeners?

### ISSUE THREAD

**Issue by IvanLi-CN** [cocopon/tweakpane#579]
Thanks to this project, I can easily add UI functionality. 

Currently I use the following code to dynamically create and delete folders：

```ts
// add

this.pane.addFolder({
    expanded: this.getRememberedExpanded(),
    // ...
})
  .on("fold", (ev) => {
     this.setRememberedExpanded( ev.expanded );
  })

// remove

this.pane.dispose();
```

From time to time I would remove the old ones and add new ones to dynamically update the interface. But every time I remove and add, presumably the hander for the fold event is not released, resulting in a memory leak. Is there any way to perform an action like `this.pane.off('fold')`?

**Comment by cocopon** [comment]
It makes sense. I'll make sure to add it to my todo list.

**Comment by cocopon** [comment]
Implemented in #587. Please wait for the next release.

**Comment by cocopon** [comment]
Released 4.0.3.
